src/common/harvest/harvest.test.js

```diff
@@ -304,9 +304,7 @@ describe('_send', () => {
 
   test('should set body to events when endpoint is events', () => {
     spec.endpoint = 'events'
-    spec.payload.body.e = {
-      [faker.datatype.uuid()]: faker.lorem.sentence()
-    }
+    spec.payload.body.e = faker.lorem.sentence()
```

Comment:
This test was the reason for the original change. However, `body.e` will always be a string so I changed the test to pass.